code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -151,6 +151,9 @@ const int CanWheelsBreakOffset = 0x893;
 const int BlinkerState = 0x899;
 
 // Wheel class
+const int WheelCamberOffset = 0x008;
```

Comment:
As seen a few lines below, is equivalent to Xrot. Why have 2 natives doing the same thing?